pkgs/by-name/co/colorized-logs/package.nix

```diff
@@ -0,0 +1,35 @@
+{ cmake
+, fetchFromGitHub
+, lib
+, stdenv
+, ...
```

Comment:
```suggestion
```

**Do not** use ellipsis

Reply:
Sorry, force of habit since the last thing I did was a NixOS module.